antora-playbook.yml

```diff
@@ -38,7 +38,7 @@ asciidoc:
     icons: font
     # for the project
     broker-ver: v3.4.0
-    building-a-custom-devfile-registry: xref:end-user-guide:making-a-workspace-portable-using-a-devfile.adoc[]
+    building-a-custom-devfile-registry: xref:end-user-guide:configuring-a-workspace-using-a-devfile.adoc[]
```

Comment:
It does not seem to have the right value, more so it's not used anywhere. So, maybe it's just better to remove it.

Reply:
Yeah, I just double-checked it. No mention in the upstream anymore. Are you giving me a green to delete it? :)

Reply:
The property is removed in https://github.com/eclipse/che-docs/pull/1737/commits/70c03801c591bdc0d6361186bc0255bd33f22b15

Does `making-a-workspace-portable-using-a-devfile.ado` really exist?

Reply:
It's probably an old name. We have quite some leftovers like this. Any help to detect and fix them is very appreciated!

Reply:
The following script:

<details>
<summary>detect_unused_properties</summary>

```bash
#!/bin/bash

# Exit on any error
set -e

# This is a configuration parameters that can be che or crwctl
PROJECT_CONTEXT=${PROJECT_CONTEXT:-che}

SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
DOCS_PROJECT_PATH=$SCRIPT_DIR/..

unused_pages=""

    readarray -d '' pages < <(cat antora-playbook.yml | yq -j '.asciidoc.attributes | keys[] | ( . + "\u0000")')

    for page in "${pages[@]}"
    do
        #`../` instead of `pages/` is used in the documentation references
        # page=${page//pages\//xref:}
        echo "Search for $page in $(pwd)"
        page="{$page}"
        if ! grep -q -r "$page" . ; then
            unused_pages="$unused_pages\n$page"
        fi
    done

if [[ "$unused_pages" ]]; then
    echo "!!! The following pages are not used in their modules."
    echo "!!! Remove them to fix the issue."
    echo -e "${unused_pages}"
    exit 1
else
    echo "All pages have references in their the modules."
fi

```

</details>

detected the following unused properties:

```
{experimental}
{favicon}
{icons}
{link-advanced-configuration-options-for-the-che-server}
{link-building-a-custom-plug-in-registry}
{link-viewing-the-state-of-the-cluster-deployment-using-openshift-4-cli-tools}
{ocp4-ver}
{prod-prev-ver-major}
{prod-url-secure}
{prod-ver-major}
```

When I will have time - I'll refactor detect_unused_images to take into account properties as well.